src/api/generate-csv/map-application-to-csv/helpers/format-currency/index.test.ts

```diff
@@ -0,0 +1,35 @@
+import formatCurrency from '.';
+import { GBP_CURRENCY_CODE } from '../../../../constants';
+
+describe('api/generate-csv/map-application-to-csv/helpers/format-currency', () => {
+  const mock = 123456;
```

Comment:
Can we please add following assertions:
* 123.123
* -123.123
* ABC

Reply:
Feels a bit overkill because we're testing the native JS `toLocaleString` behaviour, but have added these in.

No need to add a test for a string/ABC because typescript will not allow a type to be passed into this function that is not a number.